### ISSUE TITLE
EasyPal Source Code

### ISSUE THREAD

**Issue by vk6mo** [ON4QZ/QSSTV#28]
Sorry if this is off-topic. Please advise (gently) if there is a better place to put the query.

I would very much like to get hold of the Easypal source code with a view to at least doing some small fixes until a better pathway forward exists for SSTV on the Windows platform. Whilst linux is great for some of us,  there are a lot of hams out there who will only ever use Windows and, unfortuntately, they are hurting with the lack of an up-to-date windows program.

I understand Eric lost the source after some accident (?a fire) but was wondering if anyone else ever had a copy, even if it wasn't the latest version.

Chris vk3dnh, who was involved in some capacity, has suggested that someone from this qsstv group might have it, hence the query here.

If no one here can help, any suggestions on anyone else we could ask would be appreciated.

Thx

**Comment by dl8dtl** [comment]
I'm afraid this is the fate of keeping software sourcecode hidden.

It's probably easier then to find some motivated Windows developer who would be able to port QSSTV to Windows.

**Comment by vk6mo** [comment]
I agree Jorg. Porting QSSTV to Windows seems the obvious path forward.
However, if we *can* find the source, I think some little fixes would make life a lot easier for current Easypal users.

**Comment by dl8dtl** [comment]
Btw., is their DRM mode the same as QSSTV's?

**Comment by vk6mo** [comment]
Yes. Usually use B or E.
EasyPal to QSSTV decodes correctly (and vice versa).

**Comment by dl8dtl** [comment]
> I agree Jorg. Porting QSSTV to Windows seems the obvious path forward.

Then, let's see what it takes.
After all, we could setup a Github runner for compilation (of course, as well for Linux, and for MacOS).

**Comment by vk6mo** [comment]
Absolutely.
Not sure what you mean by a "Github runner" but yes if we can get the UI the same and 90% of the underlying code the same we would do really well. There's certainly bits of QSSTV that the guys using EasyPal like, so there will be incentive to use QSSTV if we can just get it on Win for them...
Assuming nothing unforeseen in the future, Im certainly happy to try to learn the win side of things and try to pull that together. To start, though, I think I would learn the code as is and do some testing, small changes for you main developers. Then step from there....
